cloudcix_primitives/nginxns.py

```diff
@@ -301,181 +212,109 @@ def scrub(
-        3114: f'3114: Failed to get `podnet_a_enabled` from config file {config_file}',
-        3115: f'3115: Failed to get `podnet_b_enabled` from config file {config_file}',
-        3116: f'3116: Invalid values for `podnet_a_enabled` and `podnet_b_enabled`, both are True',
-        3117: f'3117: Invalid values for `podnet_a_enabled` and `podnet_b_enabled`, both are False',
-        3118: f'3118: Invalid values for `podnet_a_enabled` and `podnet_b_enabled`, one or both are non booleans',
-        3119: f'3119: Failed to connect to the enabled PodNet from the config file {config_file} for find_proces_payload',
-        3120: f'3120: Failed to find process on the enabled PodNet. Payload exited with status ',
-        3121: f'3121: Failed to connect to the enabled PodNet from the config file {config_file} for stop_nginx_payload',
-        3122: f'3122: Failed to connect to the enabled PodNet from the config file {config_file} for start_nginx_payload',
-        3123: f'3123: Failed to stop nginx on the enabled PodNet. Payload exited with status ',
-        3124: f'3124: Failed to connect to the enabled PodNet from the config file {config_file} for remove_config_payload',
-        3125: f'3125: Failed to delete config file {nginx_config_path} on the enabled PodNet. Payload exited with status ',
-        3129: f'3129: Failed to run find_process_payload on the disabled PodNet. Payload exited with status ',
-        3130: f'3130: Successfully created {nginx_config_path} and started nginx on enabled PodNet '
-              f'but failed to connect to the disabled PodNet from the config file {config_file} for find_process_payload.',
-        3131: f'3131: Successfully stopped nginx and deleted {nginx_config_path} on enabled PodNet '
-              f'but failed to connect to the disabled PodNet from the config file {config_file} for stop_nginx_payload.',
-        3132: f'3132: Successfully stopped nginx and deleted {nginx_config_path} on enabled PodNet but failed to connect to the disabled PodNet '
-              f'from the config file {config_file} for stop_nginx_payload.',
-        3133: f'3133: Successfully stopped nginx and deleted {nginx_config_path} on enabled PodNet but failed to stop nginx on the disabled PodNet. '
-               'Payload exited with status ',
-        3134: f'3134: Successfully stoppend nginx and deleted {nginx_config_path} on enabled PodNet but failed to connect to the disabled PodNet '
-              f'from the config file {config_file} for remove_config_payload.',
-        3135: f'3135: Successfully stopped nginx on both PodNet nodes and deleted {nginx_config_path} on enabled PodNet but failed to stop nginx on the disabled PodNet. '
-               'Payload exited with status ',
+
+        3120: f'Failed to connect to the enabled PodNet for find_proces payload: ',
+        3121: f'Failed to run find_process payload on the enabled PodNet. Payload exited with status ',
```

Comment:
```suggestion
```

There is no error message below for `find_process` payload as if it is not running, service has already been scrubbed. 

Prefixes in messages will be correct if these suggestions are valid and applied